openshift/release/release-notes.md

```diff
@@ -0,0 +1,169 @@
+# OpenShift Pipelines 1.4
+
+# Tekton Pipelines v0.22.0
+
+## Features
```

Comment:
Waw, no "new features" in 0.22.0 ? 😅

Reply:
Yes based on this https://github.com/tektoncd/pipeline/releases/tag/v0.22.0